### PR TITLE
COG 1314 special char validation

### PR DESCRIPTION
**Pull request by kelfink**
#### Which User story does this relate to (link)?
- https://osi-cwds.atlassian.net/browse/COG-1314

#### Brief feature description
- Be more specific on validation of special chars for the password.  The list comes from https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-policies.html

Also include instruction about this list of chars in the password instructions.  Pending further deigns.

### CODE REVIEW

**Comment by kelfink** [comment]
retest this please

**Comment by sunilb0575** [comment]
The title still says WIP.

File: src/components/PasswordInstructions.test.js [hunk 1]
```diff
@@ -14,6 +15,13 @@ describe('PasswordInstructions.js Tests', () => {
     const props = {validateNumber: false, validateLowerCase: false, validateLength: false, validateUpperCase: false, validateSpecialCharacter: false}
     const wrapper = shallow(<PasswordInstructions props={props}/>)
     expect(wrapper.find('div').at(1).text()).toBe('× Minimum 8 characters× Lower case letter [a-z]× Upper case letter [A-Z]')
-    expect(wrapper.find('div').at(2).text()).toBe('× Numeric character [0-9]× Special character')
+    expect(wrapper.find('div').at(2).text()).toBe('× Numeric character [0-9]× Special character <Icon /><UncontrolledTooltip />')
+  })
+  it('should pop up to display special characters', () => {
+    const props = {validateNumber: false, validateLowerCase: false, validateLength: false, validateUpperCase: false, validateSpecialCharacter: false}
+    const div = document.createElement('div')
```
Comment (RamuKammagani-osi) [inline src/components/PasswordInstructions.test.js]: I guess we don't need this line anymore!

**Comment by sunilb0575** [comment]
The label for the cancel button on MFA page is disappearing. 
Screen Shot 2019-03-29 at 10 12 39 AM

**Comment by sunilb0575** [comment]
@kelfink I saw you fixed the label issue in MFA page, but as you have updated to the latest version of the component library, I see major changes in the UI when compared to current version UI in integration.

**Comment by kelfink** [comment]
> @kelfink I saw you fixed the label issue in MFA page, but as you have updated to the latest version of the component library, I see major changes in the UI when compared to current version UI in integration.

Yea, there have been a lot of side-effects from the upgrade.  I'll see about fixing those. Some changes may be intended by design, but the margins, for instance, seem wrong now and cause the text to overflow the login page.

**Comment by kelfink** [comment]
I made some changes to restore the original font size and realign the notice text.  The button cyan color when disabled is coming from the new version of the component library (possibly inherited from reactstrap). so I'll get feedback from Design on that part.

**Comment by kelfink** [comment]
when button is disabled:
image
when button becomes enabled:
image

**Comment by kelfink** [comment]
> The title still says WIP.

fixed

File: src/components/PasswordInstructions.test.js (hunk 1 above)
Comment (kelfink) [reply src/components/PasswordInstructions.test.js]: right!

**Comment by kelfink** [comment]
@sunilb0575 I restored a style which added that padding.

**Pull request merged**